node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/CapacityPolicies.java

```diff
@@ -82,6 +82,20 @@ private NodeResources specifiedOrDefaultNodeResources(Optional<NodeResources> re
         }
     }
 
+    private NodeResources defaultNodeResources(ClusterSpec.Type clusterType) {
+        if (zone.system() == SystemName.PublicCd && clusterType == ClusterSpec.Type.admin && zone.environment() != Environment.prod)
+            return new NodeResources(1, 3, 50);
+
+        if (zone.system() == SystemName.cd && zone.environment() == Environment.dev && zone.region().value().equals("cd-us-west-1"))
+            return new NodeResources(1, 4, 50);
+
+        if (zone.system() == SystemName.cd && zone.environment() == Environment.test || zone.environment() == Environment.staging)
+            return clusterType == ClusterSpec.Type.admin ? new NodeResources(1, 3, 50)
+                                                         : new NodeResources(4, 4, 50);
+
```

Comment:
add 0.5-3-50 default for admin for all zones - this is what we have today

Reply:
Done, thanks.